### PR TITLE
Logger and runtime load and unload 

### PR DESCRIPTION
**Pull request by stped18**
loggine core, Entity, Enemy modules 
fixing runtimen loading and unlouding mapsService , Plugins , processors,

### CODE REVIEW

File: dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/ImagePart.java [hunk 1]
```diff
@@ -15,27 +15,34 @@
...
  * @author steff
  */
-public class ImagePart implements EntityPart{
-    private Texture texture;
+public class ImagePart implements EntityPart {
+
+    private FileHandle fileHandle;
+    private int width;
+    private int hight;
+    private Texture textur = null;
```
Comment (Kjalris) [inline dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/ImagePart.java:23]: Private Texture Textur is missing an e for spelling (texture)

**Review by Kjalris — APPROVED** [review]
There is a spelling error in the Private Texture texture (textur) that needs fixing. I can run it and it works so i approve ^_^

File: dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/ImagePart.java (hunk 1 above)
Comment (stped18) [reply dk.sdu.se4.common/src/main/java/dk/sdu/se4/common/entity/part/ImagePart.java]: fix

**Comment by Nicklason** [comment]
You shouldn't be doing multiple things in one pull request. I see both logging added, format changes and changes to code logic. It would make it easier to review if it was in seperate pull requests.

**Review by Nicklason — APPROVED** [review]

**Pull request merged**